src/Http/Controllers/SwaggerLumeController.php

```diff
@@ -46,7 +46,7 @@ public function api()
         $response = new Response(
             view('swagger-lume::index', [
                 'secure' => Request::secure(),
-                'urlToDocs' => route('swagger-lume.docs'),
+                'urlToDocs' => app('url')->route('swagger-lume.docs'),
```

Comment:
If you are forcing HTTPS for all app correctly `route` should make correct url

Reply:
What this fixes is the opposite. The app is in HTTP, not HTTPS. But without the change, I get the issue mentioned here: #46.

Reply:
If so it should be configurable. it only can be an option, but not default

Reply:
Without this fix:

![image](https://user-images.githubusercontent.com/29616360/52531244-4cf8f480-2d23-11e9-9128-eed5f8ae90f3.png)

With this fix: 

![image](https://user-images.githubusercontent.com/29616360/52531209-c9d79e80-2d22-11e9-9f42-3d088f19e1df.png)

It works fine after adding the `app('url')` part.
Otherwise, not adding this, how can I configure it to work without HTTPS? I'm not forcing it.